[2024-08-11T05:30:47.977+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-11T05:30:48.023+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_migration.weather_data_migration manual__2024-08-11T05:30:46.377723+00:00 [queued]>
[2024-08-11T05:30:48.030+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_migration.weather_data_migration manual__2024-08-11T05:30:46.377723+00:00 [queued]>
[2024-08-11T05:30:48.031+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-11T05:30:48.039+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): weather_data_migration> on 2024-08-11 05:30:46.377723+00:00
[2024-08-11T05:30:48.054+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=2836) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-11T05:30:48.058+0000] {standard_task_runner.py:64} INFO - Started process 2838 to run task
[2024-08-11T05:30:48.058+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_migration', 'weather_data_migration', 'manual__2024-08-11T05:30:46.377723+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/weather_data_migration_dag.py', '--cfg-path', '/tmp/tmprntcogwk']
[2024-08-11T05:30:48.071+0000] {standard_task_runner.py:91} INFO - Job 15: Subtask weather_data_migration
[2024-08-11T05:30:48.109+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_migration.weather_data_migration manual__2024-08-11T05:30:46.377723+00:00 [running]> on host a904838c11ba
[2024-08-11T05:30:48.164+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_migration' AIRFLOW_CTX_TASK_ID='weather_data_migration' AIRFLOW_CTX_EXECUTION_DATE='2024-08-11T05:30:46.377723+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-11T05:30:46.377723+00:00'
[2024-08-11T05:30:48.166+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-11T05:30:48.167+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-11T05:30:48.167+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/weather_data_migration.py --connection postgresql+psycopg2://***:***@postgres/***']
[2024-08-11T05:30:48.179+0000] {subprocess.py:86} INFO - Output:
[2024-08-11T05:30:48.647+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-11T05:30:48.648+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-11T05:30:48.662+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_migration, task_id=weather_data_migration, run_id=manual__2024-08-11T05:30:46.377723+00:00, execution_date=20240811T053046, start_date=20240811T053048, end_date=20240811T053048
[2024-08-11T05:30:48.710+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-11T05:30:48.724+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
